### PR TITLE
Will Kwon-Intro-to-C mvp complete

### PR DESCRIPTION
**Pull request by wtkwon**
- [x]  fizzbuzz

- [x] strings

- [x] pointers

- [x] malloc

- [x] structs

- [x] queue

- [x] quicksort

### CODE REVIEW

File: fizzbuzz/fizzbuzz.c [hunk 1]
```diff
@@ -10,7 +10,19 @@
...
+            printf("FizzBuzz\n");
+        } else if (i%3==0){
+            printf("Fizz\n");
+        } else if (i%5==0) {
+            printf("Buzz\n");
+        } else {
+            counter=counter+1;
+        }
+    }
+    return counter;
```
Comment (wvandolah) [inline fizzbuzz/fizzbuzz.c]: 👍

File: pointers/pointers.c [hunk 2]
```diff
@@ -8,7 +8,9 @@
 */
 void swap(int* a, int* b)
 {
-
+    int temp=*a;
+    *a=*b;
+    *b=temp;
```
Comment (wvandolah) [inline pointers/pointers.c]: 👍

File: strings/strings.c [hunk 3]
```diff
@@ -20,7 +24,18 @@ int string_length(char s[])
 */
 char *reverse_string(char rv[], char s[])
 {
-
+    int counter=string_length(s)-1;
+    int rv_count=0;
+    while (counter>=0) {
+        rv[rv_count]=s[counter];
+        if (s[counter]==' '){
```
Comment (wvandolah) [inline strings/strings.c]: curious what this if statement is doing.  string still reverses correctly without it.

Comment (wtkwon) [reply strings/strings.c]: wouldn't work on repl unless I did that tried removing now and it works on terminal

**Pull request closed without merging**